Add finetuning job july preview contracts with maap

sagarsumant opened the pull request:
# ARM (Control Plane) API Specification Update Pull Request 

> [!TIP]
> Overwhelmed by all this guidance? See the `Getting help` section at the bottom of this PR description.

## PR review workflow diagram

Please understand this diagram before proceeding. It explains how to get your PR approved & merged.

![spec_pr_review_workflow_diagram](https://github.com/Azure/azure-rest-api-specs/assets/4429827/5bb5e7ce-8aff-4dbb-a3f8-0d9b68fef5b1)

## Purpose of this PR

What's the purpose of this PR? Check the specific option that applies. This is **mandatory**!

  - [ ] New resource provider.
  - [ ] New API version for an existing resource provider. (If API spec is not defined in TypeSpec, the PR should have been created in adherence to [OpenAPI specs PR creation guidance](https://aka.ms/azsdkdocs/createopenapispec)).
  - [ ] Update existing version for a new feature. (This is applicable only when you are revising a private preview API version.)
  - [ ] Update existing version to fix OpenAPI spec quality issues in S360.
  - [x] Other, please clarify:
    - This is adding new fields in the new API version of the existing contract for FineTuningJob to support training using user provided compute.

## Due diligence checklist

To merge this PR, you **must** go through the following checklist and confirm you understood 
and followed the instructions by checking all the boxes:

- [x] I confirm this PR is modifying Azure Resource Manager (ARM) related specifications, and not data plane related specifications.
- [x] I have reviewed following [Resource Provider guidelines](https://aka.ms/rpguidelines), including
  [ARM resource provider contract](https://aka.ms/azurerpc) and
  [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md) (estimated time: 4 hours).  
  I understand this is required before I can proceed to the diagram Step 2, "ARM API changes review", for this PR.

## Additional information

<details>
<summary> Viewing API changes</summary>

For convenient view of the API changes made by this PR, refer to the URLs provided in the table 
in the `Generated ApiView` comment added to this PR. You can use ApiView to show API versions diff. 

</details>
<details>
<summary>Suppressing failures</summary>

If one or multiple validation error/warning suppression(s) is detected in your PR, please follow the 
[suppressions guide](https://aka.ms/azsdk/pr-suppressions) to get approval.

</details>

## Getting help

- First, please carefully read through this PR description, from top to bottom. Please fill out the `Purpose of this PR` and `Due diligence checklist`.
- If you don't have permissions to remove or add labels to the PR, request `write access` per [aka.ms/azsdk/access#request-access-to-rest-api-or-sdk-repositories](https://aka.ms/azsdk/access#request-access-to-rest-api-or-sdk-repositories")
- To understand what you must do next to merge this PR, see the `Next Steps to Merge` comment. It will appear within few minutes of submitting this PR and will continue to be up-to-date with current PR state.
- For guidance on fixing this PR CI check failures, see the hyperlinks provided in given failure 
  and https://aka.ms/ci-fix.
- For help with ARM review (PR workflow diagram Step 2), see https://aka.ms/azsdk/pr-arm-review.
- If the PR CI checks appear to be stuck in `queued` state, please add a comment with contents `/azp run`.
  This should result in a new comment denoting a `PR validation pipeline` has started and the checks should be updated after few minutes.
- If the help provided by the previous points is not enough, post to https://aka.ms/azsdk/support/specreview-channel and link to this PR.

mschaofan committed:
Update description of runId to infer customer to use new GUID for create run (#29447)

* Update description of runId to infer customer to use new GUID for create run

* Refine description

GuptaVertika committed:
Guptavertika/release network microsoft.network official 2024 01 01 (#29302)

* Adds base for updating Microsoft.Network from version stable/2023-11-01 to version 2024-01-01

* Updates readme

* Updates API version in new specs and examples

* Add examples for Subnet peering and v6 Subnet peering. (#28827)

Add examples for Subnet peering and v6 Subnet peering.
---------

Co-authored-by: saurabh83 <saugu@microsoft.com>

* Init (#29127)

Co-authored-by: Abhishek Shah <shabhis@microsoft.com>

* Adding the new response code to the 2024-01-01 version release of swagger to match the bugfix in 2023-11-01 (#29048)

Co-authored-by: Arsheen Khatib (from Dev Box) <akhatib@microsoft.com>

* Added NoHealthyBackendsBehavior (#28868)

Added the property "NoHealthyBackendsBehavior" to the Probe properties.

* VirtualNetworkGatewayConnection using key vault for authentication. (#28760)

* Add parameters for cert based auth connection

* retrigger checks

* retrigger checks

* lintDiff

* Trigger Build

* revert PATCH changes

* fix

* change managed identity to v5

* add format and description

* rresolving conflict

* resolving readme file

* Revert "VirtualNetworkGatewayConnection using key vault for authentication. (#28760)"

This reverts commit c37ec243f1ed3abf88fb923eeee02d93d646a7c7.

* Lint diff error suppress

* Guptavertika/release network microsoft.network official 2024 01 01 (#29367)

* added networkidentifier under vnet->subet->serviceendpoint

* added format :armid and updated description

* corrected the type

* fixed response

* response type fixed example

---------

Co-authored-by: Bhupesh Bhatt <bhupeshbhatt@microsoft.com>

* Adding GET operaton for Network Virtual Appliance Inbound Secuirty Rule (#29339) (#29375)

* Fix for PR VirtualNetworkGatewayConnection using key vault for authentication. (#29357) (#29376)

* Fix

* Fix2

---------

Co-authored-by: Khushboo Baheti <khbaheti@microsoft.com>

* Add CrossConnection Name as an optional query parameter in List ExpressRoute CrossConnection (#29251)

* Add CrossConnection name as an optional query parameter for List CrossConnection

* update example

* Addressing comments

* Change to OData filtering

* Fix the parameter reference

* Update

---------

Co-authored-by: Jianqi Zhao <jianzhao@microsoft.com>

* initial changes (#29235)

* Changed type of networkIdentifier from PublicIPAddress to SubResource (#29469)

* added networkidentifier under vnet->subet->serviceendpoint

* added format :armid and updated description

* corrected the type

* fixed response

* response type fixed example

* Changed type of networkIdentifier from ./publicIpAddress.json#/definitions/PublicIPAddress to ./network.json#/definitions/SubResource

---------

Co-authored-by: Bhupesh Bhatt <bhupeshbhatt@microsoft.com>

---------

Co-authored-by: saurabh83 <splitt@gmail.com>
Co-authored-by: saurabh83 <saugu@microsoft.com>
Co-authored-by: Abhishek Shah <16267081+abhi7860@users.noreply.github.com>
Co-authored-by: Abhishek Shah <shabhis@microsoft.com>
Co-authored-by: AFK1194 <arsheen.khatib@gmail.com>
Co-authored-by: Arsheen Khatib (from Dev Box) <akhatib@microsoft.com>
Co-authored-by: Eugenio Peña García <70240915+EugenioPena@users.noreply.github.com>
Co-authored-by: daweiwang-microsoft <165709729+daweiwang-microsoft@users.noreply.github.com>
Co-authored-by: Bhupesh Kumar Bhatt <52891188+bhupeshbhatt1985@users.noreply.github.com>
Co-authored-by: Bhupesh Bhatt <bhupeshbhatt@microsoft.com>
Co-authored-by: Ashutosh Mishra <163998769+ashutmi@users.noreply.github.com>
Co-authored-by: Khushboo Baheti <37917868+Khushboo-Baheti@users.noreply.github.com>
Co-authored-by: Khushboo Baheti <khbaheti@microsoft.com>
Co-authored-by: jianzhao1992 <125631957+jianzhao1992@users.noreply.github.com>
Co-authored-by: Jianqi Zhao <jianzhao@microsoft.com>
Co-authored-by: nimaller <71352534+nimaller@users.noreply.github.com>

Commit:
Add finetuning job july preview contracts with maap

catalinaperalta committed:
fix endpoint (#29508)

Co-authored-by: Catalina Peralta <caperal@microsoft.com>

ChenxiJiang333 committed:
Update tspconfig.yaml (#29491)

msyyc committed:
update (#29522)

ArberH committed:
ahila devcenter Microsoft.DevCenter 2024-06-01-preview API version (#29364)

* Copy files from preview/2024-05-01-preview

Copied the files in a separate commit.
This allows reviewers to easily diff subsequent changes against the previous spec.

* Update version to preview/2024-06-01-preview

Updated the API version from preview/2024-05-01-preview to preview/2024-06-01-preview.

* Added tag for 2024-06-01-preview in readme file

* Add 2024-06-01-preview API changes

* Fix spec linting/validation issues

* add lintdiff suppression to readme for common-types sku object

* fix suppression to correct file

* move suppression to proper yaml tag

* fix filename for suppression

---------

Co-authored-by: Arber Hila <ahila@microsoft.com>
Co-authored-by: Nick Depinet <ndepinet@microsoft.com>
Co-authored-by: Nick Depinet <depinetnick@gmail.com>

v-maksymm committed:
Oracle.Database. Version 2023-09-01 stable (#29397)

* Oracle.Database. Version 2023-09-01 stable

* Oracle.Database. Fixed examples version

* Oracle.Database. Fixed examples version

* Oracle.Database. Added service-dir pramater to tspconfig

* Oracle.Database. Added service-dir pramater to tspconfig

* Oracle.Database. Rebuild after main branch merged

osmuller committed:
[AKV] Add new preview version for NSP feature (#28893)

* Start adding NSP changes to preview version

* add example

* Change to the response to be list instead of get

* Add missing example file

* Add missing enabledLogCategory

* Fix typo

* Test object type as array

* Test nsp access rule change

* Fixes for Lint

* Add pageable

* Make list not have a next link

* Add xms-mutability

* Make next link null for list

* add NextLink allthough it isnt used

* Add new preview api version and revert changes on existing one

* Create preview version from latest stable

* delete backup version

* Fix json

* Prettyfy

* fix readme

* Fix json

* update yaml (unsure of fix)

* Run prettifier

* Add get endpoint

* Change api-versions in examples

* Fix swagger

* Fix example

* Add regex for association proxy name

* prettifier

* Change names of example

* Add description to secured by perimeter

* Use common file

* Address feedback

HuShaoRu committed:
fix ComponentsByRelease for AKS 2024-04-02 preview api (#29425)

* fix definition of ComponentsByRelease

* add x-ms-identifiers

* fix

dw511214992 committed:
Change 2024-05-01 to 2024-05-01-preview (#29441)

* rename 2024-05-01 to 2024-05-01-preview

* [HdiOnAks] Adding PublicIpTag property to ClusterPoolNetworkProfile (#29497)

* Adding PublicIpTag property to ClusterPoolNetworkProfile

---------

Co-authored-by: Yuan He <yuanhe@outlook.com>

---------

Co-authored-by: Wei Dong <weidong@microsoft.com>
Co-authored-by: Yuan He <yh772@cornell.edu>
Co-authored-by: Yuan He <yuanhe@outlook.com>

raych1 committed:
Create sdk-automation-faq.md (#29537)

* Create sdk-automation-faq.md

Added a document for FAQ as placeholder
Added instructions of downloading SDK

* Update sdk-automation-faq.md

Added short link

* Update sdk-automation-faq.md

Added blank lines before/after screenshot.

bjqian committed:
Add new api version 2024-04-01-preview for Azure SignalR and Azure Web PubSub (#29239)

* Init 2024-04-01-preview from 2023-03-01

* Add Api version 2024-04-01-preview

* Update to 2024-04-01-preview for web pubsub

kazrael2119 committed:
disable flatten for JS for mongocluster, deploymentstacks and appcomplianceautomation  (#29361)

* disable flatten for JS for mongocluster

* Update readme.typescript.md

* update

stevemunk committed:
Update documentation for Azure Maps Route v1 API (#29524)

* https://msazure.visualstudio.com/One/_workitems/edit/28479372

* https://msazure.visualstudio.com/One/_workitems/edit/28479372

nikhilpadhye1 committed:
Moving missed changes from 2023-11-01 packet capture swagger fix to l… (#29473)

* Adds base for updating Microsoft.Network from version stable/2023-11-01 to version 2024-01-01

* Updates readme

* Updates API version in new specs and examples

* Add examples for Subnet peering and v6 Subnet peering. (#28827)

Add examples for Subnet peering and v6 Subnet peering.
---------

Co-authored-by: saurabh83 <saugu@microsoft.com>

* Init (#29127)

Co-authored-by: Abhishek Shah <shabhis@microsoft.com>

* Adding the new response code to the 2024-01-01 version release of swagger to match the bugfix in 2023-11-01 (#29048)

Co-authored-by: Arsheen Khatib (from Dev Box) <akhatib@microsoft.com>

* Added NoHealthyBackendsBehavior (#28868)

Added the property "NoHealthyBackendsBehavior" to the Probe properties.

* VirtualNetworkGatewayConnection using key vault for authentication. (#28760)

* Add parameters for cert based auth connection

* retrigger checks

* retrigger checks

* lintDiff

* Trigger Build

* revert PATCH changes

* fix

* change managed identity to v5

* add format and description

* added networkidentifier under vnet->subet->serviceendpoint (#29292)

* added networkidentifier under vnet->subet->serviceendpoint

* resolving conflict (#29343)

* Fix for PR VirtualNetworkGatewayConnection using key vault for authentication. (#29357)

* Fix

* Fix2

---------

Co-authored-by: Khushboo Baheti <khbaheti@microsoft.com>

* Adding GET operaton for Network Virtual Appliance Inbound Secuirty Rule (#29339)

* Moving missed changes from 2023-11-01 packet capture swagger fix to latest api version

* Add CrossConnection Name as an optional query parameter in List ExpressRoute CrossConnection (#29251)

* Add CrossConnection name as an optional query parameter for List CrossConnection

* update example

* Addressing comments

* Change to OData filtering

* Fix the parameter reference

* Update

---------

Co-authored-by: Jianqi Zhao <jianzhao@microsoft.com>

* initial changes (#29235)

* adding supression for the breaking change in azure-sdk-for-go

* adding correct package for this supression

---------

Co-authored-by: GuptaVertika <149072034+GuptaVertika@users.noreply.github.com>
Co-authored-by: saurabh83 <splitt@gmail.com>
Co-authored-by: saurabh83 <saugu@microsoft.com>
Co-authored-by: Abhishek Shah <16267081+abhi7860@users.noreply.github.com>
Co-authored-by: Abhishek Shah <shabhis@microsoft.com>
Co-authored-by: AFK1194 <arsheen.khatib@gmail.com>
Co-authored-by: Arsheen Khatib (from Dev Box) <akhatib@microsoft.com>
Co-authored-by: Eugenio Peña García <70240915+EugenioPena@users.noreply.github.com>
Co-authored-by: daweiwang-microsoft <165709729+daweiwang-microsoft@users.noreply.github.com>
Co-authored-by: Bhupesh Kumar Bhatt <52891188+bhupeshbhatt1985@users.noreply.github.com>
Co-authored-by: Khushboo Baheti <37917868+Khushboo-Baheti@users.noreply.github.com>
Co-authored-by: Khushboo Baheti <khbaheti@microsoft.com>
Co-authored-by: Ashutosh Mishra <163998769+ashutmi@users.noreply.github.com>
Co-authored-by: jianzhao1992 <125631957+jianzhao1992@users.noreply.github.com>
Co-authored-by: Jianqi Zhao <jianzhao@microsoft.com>
Co-authored-by: nimaller <71352534+nimaller@users.noreply.github.com>

ajtms committed:
Fixes for x-ms-examples used in 2023 API specification validation (#29452)

* Added missing id property to some example resources. Added headers with locations to long-running operation examples.

* Added some missing information on some response object typings regarding the location property used in long-running operations.

* Fixed a possible typo 'azure-async-operation' to 'azure-asyncoperation' (removed a hyphen).

* Ran prettier against azureFirewall.json

* Applying the 2023-05-01 validation fixes to the 2023-06-01 and 2023-09-01 versions as well.

* Removed duplicate property

* Added missing location header to the 06 and 09 verisions, similar to the 05 version.

* Added missing IDs to certain resource gets/puts. Added missing location header for long running delete operation.

* Updated 2023-09-01 stable examples.

* Fixed improper resource id

* Fixed example errors for 2023-06-01. Fixed small whitespace typo on a 2023-09-01 file.

* Updated 2023-05-01 firewall examples. Fixed small whitespace typoe missed from previous commit.

* Updated 2023-04-01 examples with resource IDs. Updated some resource IDs that use the wrong firewall name in their path.

* Updated firewall examples for 2023-02-01.

* Added location headers for several long-running operations. Updated some location headers so their URL paths have the accurate api-version.

ChenglongLiu committed:
Fix secret source url format (#29475)

msyyc committed:
[powerbiembedded] Update readme.md (#29551)

* Update readme.md

* Create readme.python.md

v-hongli1 committed:
Added `stream-style-serialization: false` in `readme.java.md`. (#29542)

* Added `stream-style-serialization: false` in `readme.java.md`.

weidongxu-microsoft committed:
network, java directive (#29552)

raych1 committed:
Update sdk automation document (#29553)

remove openapi hub related steps;
change breaking change label property name.

almasak committed:
Make "marketplaceSubscriptionId" an optional field for "MarketplaceDetails" (#29541)

badeamarjieh committed:
Introduce Microsoft Fabric ARM Rest APIs (#29355)

* 2023-11-01

* .

* fix json file name

* update resume and suspend operations

* update examples of resume and suspend

* fixed json format

* resolve comments

* .

* .

* .

* update description

* updating State model to ResourceState

* fix comments

xiaoxuqi-ms committed:
add new GA version: 2023-12-30 for Microsoft.DBForMySQL (#29031)

* add base commit by copying previous version

* update api version

* add suppression yaml

* add suppression yaml

* fix suppression yaml

* move suppression.yaml to service folder

* fix suppression

* fix path

* remove migration

kristapratico committed:
fix docs creating bad docstrings in purview (#29558)

* fix docs creating bad docstrings in purview

* fix ci

ramoka178 commented:
>                 "subscriptionId": {

Use from common types v5 ?

---
Refers to: specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2024-07-01-preview/mfe.json:12510 in d15dc2a. [](commit_id = d15dc2ae2488996aa7670d3fd5d5a23e1fa9d83e, deletion_comment = False)

ramoka178 commented:
>                             "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"

v5 is the current and recommended version. Please upgrade to it all through the file.

---
Refers to: specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2024-07-01-preview/mfe.json:49 in d15dc2a. [](commit_id = d15dc2ae2488996aa7670d3fd5d5a23e1fa9d83e, deletion_comment = False)

ramoka178 commented:
>                 "resourceGroup": {

Use from common types v5 ?

---
Refers to: specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2024-07-01-preview/mfe.json:12515 in d15dc2a. [](commit_id = d15dc2ae2488996aa7670d3fd5d5a23e1fa9d83e, deletion_comment = False)

ramoka178 commented:
>                         "description": "Random forest is a supervised learning algorithm.\nThe \"forest\" it builds, is an ensemble of decision trees, usually trained with the â€œbaggingâ€ method.\nThe general idea of the bagging method is that a combination of learning models increases the overall result."

fix it ?

---
Refers to: specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2024-07-01-preview/mfe.json:13467 in d15dc2a. [](commit_id = d15dc2ae2488996aa7670d3fd5d5a23e1fa9d83e, deletion_comment = False)

ramoka178 commented:
>         "CustomModelFineTuning": {

where is this used ?

---
Refers to: specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2024-07-01-preview/mfe.json:14298 in d15dc2a. [](commit_id = d15dc2ae2488996aa7670d3fd5d5a23e1fa9d83e, deletion_comment = False)

ramoka178 commented:
>         "FineTuningJob": {

Where is this used?

---
Refers to: specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2024-07-01-preview/mfe.json:16599 in d15dc2a. [](commit_id = d15dc2ae2488996aa7670d3fd5d5a23e1fa9d83e, deletion_comment = False)

ramoka178 commented:
>                         "description": "Random forest is a supervised learning algorithm.\nThe \"forest\" it builds, is an ensemble of decision trees, usually trained with the â€œbaggingâ€ method.\nThe general idea of the bagging method is that a combination of learning models increases the overall result."

fix it?

---
Refers to: specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2024-07-01-preview/mfe.json:16937 in d15dc2a. [](commit_id = d15dc2ae2488996aa7670d3fd5d5a23e1fa9d83e, deletion_comment = False)

ramoka178 commented:
1. Please add a base commit, with existing api version and then keep adding you changes.
2. Fix the conflicts
3. Some definitions in the mfe.json are not used
4. the examples are not referenced.


I dont think the PR is yet to reviewed. Please fix all these things and place it back in the queue.

bavneetsingh16 committed:
Microsoft.Kubernetes to add preview version 2024-07-01-preview (#29487)

* copy API version from 2024-06-01-preview to 2024-07-01-preview

* updating API version to 2024-07-01-preview

* update readme files

* add configurationSettings parameter for this api version

* add secret flag for protectedSettings

* update readme file

* made gateway as first class param

* update to a more appropriate name

* fix prettier issue

---------

Co-authored-by: Bavneet Singh <bavneetsingh@microsoft.com>

Y-Sindo committed:
Add Web PubSub data plane version 2024-01-01 (#29445)

* Add Web PubSub data plane version 2024-01-01

* Add Web PubSub data plane version 2024-01-01

---------

Co-authored-by: Liangying.Wei <lianwei@microsoft.com>

mitsha-microsoft committed:
Added 2023-12-01 preview API for Azure Load Testing Control Plane (#29543)

* Added 2023-12-01 API Specs and Initial Examples

* Added validated examples

* Changes in the README and Provisioning State Suppression

* Format changes

* Pattern on resource names

kristapratico committed:
fix docs creating bad docstrings in healthinsights (#29557)

* fix docs creating bad docstrings in healthinsights

* fix ci

tbyfield committed:
Microsoft.DevCenter 2024-07-01-preview (#29527)

* Copy files from preview/2024-06-01-preview

Copied the files in a separate commit.
This allows reviewers to easily diff subsequent changes against the previous spec.

* Update version to preview/2024-07-01-preview

Updated the API version from preview/2024-06-01-preview to preview/2024-07-01-preview.

* Added tag for 2024-07-01-preview in readme file

* Change introduces a new feature in the form of an API for Imaging on top of Image Definition.

* Add suppression same as previous API version

* Fixed example. Ran prettier

* Remove suppression

* re-add suppression

* Update references to commontypes to v5 for changes included in this PR

* Per PR comment, renamed getDetails to getBuildDetails. Added Azure-AsyncOperation as header returned in new async post actions

* Generated new example subscription id as the previous one was not passing the uuid validation locally

* Remove unused ListImageBuildTaskGroupsIncludeParameter

---------

Co-authored-by: Nick Depinet <depinetnick@gmail.com>

srnagar committed:
Text Translation: Enable stream style serialization (#29509)

msyyc committed:
Update readme.python.md (#29587)

* Update readme.python.md

* Create readme.python.md

golddove committed:
[StorageTasks] Bugfix maxpagesize type (#29534)

* [StorageTasks] Bugfix maxpagesize type

* add sdk-suppressions.yaml

---------

Co-authored-by: Alancere <804873052@qq.com>

bradenwatkins committed:
Billing/2024 04 01 (#28160)

* Baseline for 2024-04-01 from 2020-05-01

* Updates for GA 2024-04-01

* Adding suppression as 'resource' schema is missing properties

* Build fixes and PR feedback

* tr lint error fix

* update example

* try fixing python

* try removing the override

* updating invoice download uri

* Update reservation.json

* Removing additional properties on policy

* updated the resource reference and removed unused override

* fixed error response

* fix error

* updated to proxyResourcewithTags

* fix errors

* Prettier check fix

* merge conflict resolved

* Prettier fix

* reverted package.json

* updated the ref

* Prettier check

* updated operation.json

* add sdk-suppressions.yaml

* remove modelerfour.lenient-model-deduplication, first stable

* fix sdk-suppressions

---------

Co-authored-by: Braden Watkins (from Dev Box) <brwatkin@microsoft.com>
Co-authored-by: Umang Shah <umangshah@microsoft.com>
Co-authored-by: us6193 <umangshah6193@gmail.com>
Co-authored-by: gaoyp830 <51546332+gaoyp830@users.noreply.github.com>
Co-authored-by: Alancere <804873052@qq.com>

drishtipeshwani09 committed:
Microsoft.ContainerInstance: Added API version 2024-05-01-preview (#28964)

* Copy files from stable/2023-05-01

Copied the files in a separate commit.
This allows reviewers to easily diff subsequent changes against the previous spec.

* Update version to preview/2024-05-01-preview

Updated the API version from stable/2023-05-01 to preview/2024-05-01-preview.

* Added tag for 2024-05-01-preview in readme file

* Added ContainerGroupProfile, StandbyPoolProfile, isCreatedFromStandbyPool to Container Group Properties and ConfigMap to ContainerProperties specs and modified examples

* Corrected code style issues using prettier

* Fixed code styling issues using prettier

* Corrected definition types

* Added isCreatedFromStandbyPool to container group properties in examples

* Corrected config map property definition in example

* Added arm resource id identifier for container group profile and standby pool profile

* Added format variable for container group profile and standby pool profile reference id

* Added 200 http status response in standby pool scenario example

* Modified the Container Group Create with standby pools example

* Corrected format

* Created suppressions.yaml

* Update containerInstance.json to remove osType,image and resources as required properties

* Update suppressions.yaml

* Update suppressions.yaml

* Delete specification/containerinstance/resource-manager/suppressions.yaml

* Added suppression for AvoidAdditionalProperties in readme.md

* Update suppression in readme.md

* Update readme.md

iscai-msft committed:
use body root to specify which parameter models we want to keep grouped (#29483)

* add ops.ResourceCollectionAction KeepModel

* add ops.ResourceAction KeepModel

* add ops.LongRunningResourceAction KeepModel

* other templates

* switch over to BodyParameter

* format

* use valueof in BodyParameter template, add doc

* remove unnecessary documentation

* add doc to BodyParameter template

* add extra specs

* fix formatting of BodyParameter

* format

* format

* format

* suppress doc warning

* move body parameter to template

* add default body parameter doc

* default bodyparam doc to body parameter

* fix validation errors

* remove bodyparameter from codetransparency

* try for openai

* remove incorrect alias bodyparameter

* don't spread CreateLivenessWithVerifySessionContent

* fix breaking change in easm

* add readme to openai assistants

* update readme

* add tag to readme

* fix json file reference

* add json files to list

---------

Co-authored-by: iscai-msft <isabellavcai@gmail.com>
Co-authored-by: Mark Cowlishaw <markcowl@microsoft.com>

yaotongms committed:
hybridcompute - update readme.md and description for SDK commands generation (#29510)

* add back command

* remove run command PATCH

* update operation description

mikeharder committed:
Move TypeSpec content from ci-fix.md to wiki (#29600)

mbelt committed:
Add Azure async APIs to Maps data plane api-version 2023-12-01-preview (#29472)

* Add Azure async APIs.

* Add examples, fix return codes.

* Fix typo.

* Add tags, run linter.

* Add missing refs.

* Formatting

* Add missing parameters.

* Remove unused status codes, defined Location header.

* Model validation errors.

* Prettier

* ProvisioningStateSpecifiedForLROPut

* Address PR comments.

* Create sdk-suppressions.yaml

---------

Co-authored-by: Marcus Beltman <mabeltma@microsoft.com>
Co-authored-by: Yuchao Yan <yuchaoyan@microsoft.com>

elizatargithub7 committed:
Quotas for DevOpsInfrastructure (#29581)

* up

* up

* almost there

* example

* up

* up

* up

* review comments

alzimmermsft committed:
Revert WebApiSkill design change (#29614)

rakshithtr committed:
Fixing swagger model validation failures (#29507)

* Fixing swagger model validation failures

* Added headers and changed final state via

* Adding location also as part of header

v-maksymm committed:
Oracle.Database. Removed readonly properties from required (#29618)

* Oracle.Database. Readonly properties not required anymore

* Oracle.Database. Readonly properties not required anymore

MaryGao committed:
tsp, loadtesting, update client.tsp to fix the path change issue for action operation (#29597)

* Update the client.tsp

* Update specification/loadtestservice/LoadTestService/client.tsp

ArthurMa1978 committed:
[Informatica] Fix property type (#29605)

weidongxu-microsoft committed:
java, informatica disable emitter in tspconfig (#29598)

mikeharder committed:
[TypeSpecRequirement] Convert to GitHub Action (#29615)

YunsongB committed:
Add AOAI 2024-06-01 GA API version; (#29612)

Co-authored-by: Yunsong Bai <yunba@microsoft.com>
Co-authored-by: Travis Wilson <35748617+trrwilson@users.noreply.github.com>

Commit:
Fix the swagger

Commit:
Dummy change.

sagarsumant committed:
Merge branch 'Azure:main' into sasum/finetuning-job-with-maap
